server/service/appconfig.go

```diff
@@ -644,6 +644,13 @@ func (svc *Service) validateMDM(
 		}
 	}
 
+	if !mdm.WindowsEnabledAndConfigured {
+		if mdm.WindowsSettings.CustomSettings.Set && len(mdm.WindowsSettings.CustomSettings.Value) != len(oldMdm.WindowsSettings.CustomSettings.Value) {
```

Comment:
Same validation here, whatever we decide to do for the team's custom settings.

Reply:
for future reference, we're continuing the conversation here https://github.com/fleetdm/fleet/pull/15145#discussion_r1393264552